### PR TITLE
build!: update library to use Node 12

### PR DESCRIPTION
**Pull request by sofisl**
Migrate library to Node 12

### CODE REVIEW

**Comment by dnoe** [comment]
@aabmass Can you take a look? Let me know if you haven't had the approver bit turned on for your user yet and you need me to stamp this.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14, 16]
+        node: [12, 14, 16]
```
Comment (aabmass) [inline .github/workflows/ci.yaml:12]: Should we add node 18 as well since it's current?

**Review by aabmass — COMMENTED** [review]
Do recommend we make a major version release of this repo after this?

In addition to removing official support for node10, I know some TS major version bumps can produce incompatibilities in the compiled output

**Review by aabmass — APPROVED** [review]
LGTM

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (sofisl) [reply .github/workflows/ci.yaml:12]: It's not yet stable, but we will do this once we migrate to 14!

**Pull request merged**